build.sbt

```
@@ -339,6 +339,11 @@ lazy val `soft-opt-in-consent-setter` = lambdaProject(
   Seq(awsSecretsManager, circe, circeParser, scalatest, scalajHttp, awsS3, simpleConfig) ++ logging
 ).dependsOn(`effects-s3`, `effects-cloudwatch`, `salesforce-core`)
 
+lazy val `sf-emails-to-s3-exporter` = lambdaProject(
+  "sf-emails-to-s3-exporter",
+  "Runs regularly to retrieve emails from Salesforce and save as json in S3",
+  Seq(awsSecretsManager, circe, circeParser, scalajHttp, awsS3))
```

Are you actually using `awsSecretsManager` programmatically?  If it's just being used to find env vars at lambda install time, you don't need it as a dependency.
```suggestion
  Seq(circe, circeParser, scalajHttp, awsS3))
```

---

Done